public/actions/createCovercryptKeyPair.java

```diff
@@ -0,0 +1,8 @@
+//
+// Create Covercrypt Key Pair
+//
+public static void createCoverCryptMasterKeyPair(String kmsServerUrl, String apiKey) {
+  Policy policy = policy();
+  KmsClient kmsClient = new KmsClient(kmsServerUrl, apiKey);
```

Comment:
the actual `KmsClient` constructor is:
```
    public KmsClient(String server_url, Optional<String> api_key) {
        this(new RestClient(server_url, api_key));
    }
```

`apiKey` should be an `Optional<String>`